core/build.xml

```diff
@@ -14,70 +14,73 @@
     <subant buildpath="methods" target="dist" />
   </target>
 
-  <target name="compile" description="Compile" depends="methods-build">
-    <taskdef name="methods" 
-	     classname="PAppletMethods" 
-	     classpath="methods/methods.jar" />
-    <methods dir="${basedir}/src/processing/core" recorder="true" />
+  <macrodef name="compilecommon">
+    <attribute name="destdir"/>
+    <attribute name="srcdir"/>
+    <sequential>
+      <taskdef name="methods"
+  	     classname="PAppletMethods"
+  	     classpath="methods/methods.jar" />
+      <methods dir="${basedir}/src/processing/core" recorder="true" />
+
+      <!-- Where can I expect to find Java Mode JARs? -->
+      <property name="java.mode" value="../java/mode/" />
+
+      <!-- JavaFX got removed from the Oracle's JDK for arm -->
+      <condition property="fx.unavailable" value="true">
```

Comment:
[preliminary self-review] It is probably wise to expand this to beyond arm given status of OpenJFX. Possibly edit README for build instructions.

Reply:
Brought something similar to build/build.xml to allow for building Processing even if OpenJFX is not installed. Will further discuss in updated build instructions.